test/html.test.js

```diff
@@ -665,3 +672,88 @@ QUnit.test('does not wrap mp4 source buffers', function() {
   );
   QUnit.equal(mediaSource.sourceBuffers.length, 2, 'created native buffers');
 });
+
+QUnit.test('can get activeSourceBuffers', () => {
+  let mediaSource = new videojs.MediaSource();
+
+  QUnit.ok(Array.isArray(mediaSource.activeSourceBuffers));
```

Comment:
*Technically* active source buffers should be a SourceBufferList so it seems weird to use `Array.isArray` here. What about testing for `length` or something else array-ish?